crates/amaru/src/bin/amaru/main.rs

```diff
@@ -63,6 +63,9 @@ enum Command {
 
     /// Import VRF nonces intermediate states
     ImportNonces(cmd::import_nonces::Args),
+
+    /// Dump the content of the chain database
```

Comment:
Perhaps add a bit more details about the command, eg. that it's meant for troubleshooting purposes and dumps the _whole_ content of the DB which might be potentially huge, in hex encoded values?

Reply:
Good idea, done.